src/django/api/views.py

```diff
@@ -307,6 +307,20 @@ def all_countries(request):
     return Response(COUNTRY_CHOICES)
 
 
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def facilities_count(request):
```

Comment:
Consider moving this under the `FacilitiesViewSet` to so we can `GET /api/facilities/count/` without adding an explicit URL.

```python
@action(detail=False, methods=['get'])
def count(self, request):
    """
    Returns a count of total Facilities available in the Open Apparel
    Registry.

    ### Sample Response
       { "count": 100000 }

    """
    count = Facility.objects.count()
    return Response({"count": count})
```

Reply:
Cool, that is definitely better:

![Screen Shot 2019-03-22 at 2 21 13 PM](https://user-images.githubusercontent.com/4165523/54844656-c6362080-4cad-11e9-82cd-5b1ff18b0e2e.png)

![Screen Shot 2019-03-22 at 2 21 13 PM](https://user-images.githubusercontent.com/4165523/54844676-ccc49800-4cad-11e9-837d-b47198a605dd.png)

I will swash the fixup together with the two commits here.